packages/sensor/lib/sensor.dart

```diff
@@ -0,0 +1,36 @@
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+
+const EventChannel _accelerometerEventChannel =
+    const EventChannel('plugins.flutter.io/accelerometer');
+
+const EventChannel _gyroscopeEventChannel =
+    const EventChannel('plugins.flutter.io/gyroscope');
+
+Stream<List<double>> _accelerometerEvents;
+Stream<List<double>> _gyroscopeEvents;
+
+List<double> numToDouble(List<num> l) {
+  return l.map((num v) => v.toDouble());
+}
+
+/// A broadcast stream of events from the device accelerometer.
```

Comment:
Can you add more docs on what the stream events mean (how to interpret the data)?